_topic_maps/_topic_map.yml

```diff
@@ -3086,6 +3086,11 @@ Topics:
     Topics:
     - Name: Backing up applications on ROSA STS using OADP
       File: oadp-rosa-backing-up-applications
+  - Name: OADP and AWS STS
+    Dir: aws-sts
+    Topics:
+    - Name: Backing up applications on AWS STS using OADP
+      File: oadp-aws-sts     	
```

Comment:
![image](https://github.com/openshift/openshift-docs/assets/106804821/38052ad0-82a5-415c-a646-57c0afc3f8e7)
@CarmiWisemon - If you look at the image you can see you have some white-spaces (shown by `.` s) and a tab or `\t` (shown by `-`)
If you delete those white-spaces and the tab, you will be good to go

Reply:
@anarnold97 
Thank you and thank you for all of your help!!!!!